src/sagemaker/tensorflow/estimator.py

```diff
@@ -116,13 +117,16 @@ def __init__(self, training_steps=None, evaluation_steps=None, checkpoint_path=N
             checkpoint_path (str): Identifies S3 location where checkpoint data during model training can be
                 saved (default: None). For distributed model training, this parameter is required.
             py_version (str): Python version you want to use for executing your model training code (default: 'py2').
+            requirements_file (str): Path to a ``requirements.txt`` file (default: ''). The path should be relative to
```

Comment:
I believe the path should be within the source_dir. By saying relative a user could do:

requirements_file="../requirements_file.txt"

but it wouldn't have been uploaded. Then it will fail.

Reply:
good point - clarified the wording